website/source/docs/job-specification/update.html.md

```diff
@@ -22,11 +22,11 @@ description: |-
 </table>
 
 The `update` stanza specifies the group's update strategy. The update strategy
-is used to control things like rolling upgrades and canary deployments. If
-omitted, rolling updates and canaries are disabled. If specified at the job
-level, the configuration will apply to all groups within the job. If multiple
-`update` stanzas are specified, they are merged with the group stanza taking the
-highest precedence and then the job.
+is used to control things like [rolling upgrades][rolling] and [canary
+deployments][canary]. If omitted, rolling updates and canaries are disabled. If
+specified at the job level, the configuration will apply to all groups within
+the job. If multiple `update` stanzas are specified, they are merged with the
+group stanza taking the highest precedence and then the job.
```

Comment:
Do you also want to expand on how task groups are updated individually and there isn't coordination between them.